implementation/libs/jbb-lib-properties/src/main/java/org/jbb/lib/properties/FreshInstallPropertiesCreator.java

```diff
@@ -34,14 +50,43 @@ private static void getDefaultFromClasspath(File propertyFile) {
         }
     }
 
+    private static PropertiesConfiguration getReferenceProperties(File propertyFile) {
```

Comment:
What is referenceproperties? In your opinion this method name is correct? 
To be honest I dont know what do you mean by that, where I can find reference properties file :(